extensions/wikia/ArticleExporter/ArticleExporterHooks.class.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Wikia\Rabbit\ConnectionBase;
+
+/**
+ * @author Lore team
+ *
+ * Send plaintext versions of articles to RabbitMQ as they are edited on Fandom
+ */
+
+class ArticleExporterHooks {
+    const ROUTING_KEY = "taxonomy.article-edits";
```

Comment:
The rest of the queues we've been using have been singular. Can we change this?

Also minor: it'd be nice if this was configureable so that like the rest of our flows we can choose to have a personalized queue here. What do you think?